pkgs/applications/virtualization/distrobox/default.nix

```diff
@@ -2,19 +2,22 @@
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "distrobox";
-  version = "1.5.0.2";
+  version = "1.6.0.1";
 
   src = fetchFromGitHub {
     owner = "89luca89";
     repo = finalAttrs.pname;
     rev = finalAttrs.version;
-    hash = "sha256-ss8049D6n1V/gDzEMjywDnoke5s2we9j3mO8yta72UA=";
+    hash = "sha256-UWrXpb20IHcwadPpwbhSjvOP1MBXic5ay+nP+OEVQE4=";
   };
 
   dontConfigure = true;
   dontBuild = true;
 
   nativeBuildInputs = [ makeWrapper ];
+
+  patches = [ ./always-mount-nix.patch ];
```

Comment:
It's a good idea to reference the upstream patch here, to make the tracking easier.

Reply:
Added a reference to the upstream PR